src/main/java/com/newworld/saegil/simulator/conversation/controller/AudioConversationController.java

```diff
@@ -0,0 +1,34 @@
+package com.newworld.saegil.simulator.conversation.controller;
+
+import java.util.Map;
+
+import org.springframework.core.io.buffer.DataBuffer;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.newworld.saegil.simulator.conversation.service.AudioConversationService;
+
+import lombok.RequiredArgsConstructor;
+import reactor.core.publisher.Flux;
+
+@RestController
+@RequestMapping("/api/v1/stt")
+@RequiredArgsConstructor
+public class AudioConversationController {
+
+    private final AudioConversationService audioConversationService;
+
+    @PostMapping(value = "/audio-to-audio", consumes = MediaType.APPLICATION_OCTET_STREAM_VALUE, produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
+    public Flux<DataBuffer> audioConversation(@RequestBody byte[] audioFileBytes) {
```

Comment:
어노테이션 속성 쪽도 개행을 하면 좀 더 직관적이에요.
파라미터에 final 선언도 추천합니다.

```suggestion
    @PostMapping(
            value = "/audio-to-audio", 
            consumes = MediaType.APPLICATION_OCTET_STREAM_VALUE, 
            produces = MediaType.APPLICATION_OCTET_STREAM_VALUE
    )
    public Flux<DataBuffer> audioConversation(@RequestBody final byte[] audioFileBytes) {
```